biggraphite/cli/command_copy.py

```diff
@@ -95,7 +110,8 @@ def run(self, accessor, opts):
             self._copy_metric(accessor, src_metric, dst_metric, time_start, time_end)
 
     @staticmethod
-    def _get_metric_tuples(accessor, src, dst, recursive, dry_run=True):
+    def _get_metric_tuples(accessor, src, dst, recursive,
+                           src_retention, dst_retention, dry_run=True):
```

Comment:
Maybe put
src_retention = bg_accessor.Retention.from_string(src_retention)
dst_rerention = bg_accessor.Retention.from_string(dst_retention) 

You will have less work to do in the loop